packages/appstore/src/components/wallet-content/wallet-transfer-block.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Button, Text } from '@deriv/components';
+import { formatMoney } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize } from '@deriv/translations';
+import CurrencySwitcherContainer from 'Components/containers/currency-switcher-container';
+import { TWalletAccount } from 'Types';
+
+type TProps = {
+    wallet_account: TWalletAccount;
+};
+
+const WalletTransferBlock = observer(({ wallet_account }: TProps) => {
+    const { traders_hub, ui } = useStore();
+    const { setIsWalletModalVisible } = ui;
+    const { setWalletModalActiveWalletID, setWalletModalActiveTab } = traders_hub;
+
+    const { currency, balance, loginid } = wallet_account;
+
+    return (
+        <CurrencySwitcherContainer
+            title={null}
```

Comment:
this prop type should be made optional in CurrencySwitcherContainer, so that we don't have to pass null, please update or leave a TODO comment